src/client/app/admin/authentication/authentication.service.ts

```diff
@@ -50,7 +51,7 @@ export class AuthenticationService {
 	};
 
 	// Get the user who is currently logged in (or null if no one is logged in)
-	public getCurrentUser(): User {
+	public getCurrentUser() {
```

Comment:
Dunno if my comment showed up in the other thread, but I'd propose you at least make this return 'any' for now.